SPRCLI/Sources/SPR/Configurable.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ClientRuntime
+import Foundation
+import AWSCLIUtils
+
```

Comment:
The protocol & extension below lets us share some config code between the two Registry CLI commands.